app/views/all-components.njk

```diff
@@ -19,9 +19,7 @@
       All components
     </h1>
   </div>
-  {% for data in componentData %}
-    {% set componentName = data.componentName %}
-    {% set componentNameHuman = componentName | replace("-", " ") %}
-    {{ showExamples(componentName, componentNameHuman, data, legacyQuery) }}
+  {% for component in components %}
+    {{ showExamples(component, legacyQuery) }}
```

Comment:
To remove a loop somewhere I added `.name` to each component. Was helpful here too